setup.py

```diff
@@ -93,7 +93,9 @@ def __init__(self):
         super(Protobuf, self).__init__()
         # TODO: allow user specify protobuf include_dirs libraries with flags
         # and environment variables
-        self.libraries = ['protobuf']
+        assert os.getenv('PROTOBUF_LIB')
+        self.libraries = [os.path.join(os.getenv('PROTOBUF_LIB'), "lib\\libprotobuf")]
```

Comment:
`\` is not cross platform. `os.path.join` can handle this if you split it into separated parts.
Shared library suffixes are different on varies platforms (.e.g `.so`, `.dylib` ...), so it's better to let distutils/setuptools to handle it for you. (i.e. setting `self.libraries`, `self.lib_dirs`and `self.rpaths`).

Reply:
Make sense! Thank you!

Fixed it accordingly.